installer/win/util/brave_vpn_helper_utils.cc

```diff
@@ -1,25 +1,42 @@
-/* Copyright (c) 2023 The Brave Authors. All rights reserved.
- * This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this file,
- * You can obtain one at https://mozilla.org/MPL/2.0/. */
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
 
-#include "brave/components/brave_vpn/browser/connection/ikev2/win/brave_vpn_helper/brave_vpn_helper_state.h"
+#include "brave/installer/win/util/brave_vpn_helper_utils.h"
 
-#include <windows.h>
-
-#include "base/files/file_path.h"
-#include "base/logging.h"
+#include "base/path_service.h"
+#include "base/process/launch.h"
 #include "base/strings/utf_string_conversions.h"
 #include "base/win/registry.h"
-#include "brave/components/brave_vpn/browser/connection/ikev2/win/brave_vpn_helper/brave_vpn_helper_constants.h"
+#include "base/win/scoped_handle.h"
 #include "brave/components/brave_vpn/common/brave_vpn_utils.h"
 #include "brave/components/brave_vpn/common/win/scoped_sc_handle.h"
 #include "brave/components/brave_vpn/common/win/utils.h"
+#include "brave/components/brave_vpn/common/wireguard/win/service_constants.h"
+#include "brave/components/brave_vpn/common/wireguard/win/service_details.h"
+#include "brave/components/brave_vpn/common/wireguard/win/wireguard_utils_win.h"
+#include "brave/installer/win/util/brave_vpn_helper_constants.h"
 #include "chrome/install_static/install_modes.h"
 #include "chrome/install_static/install_util.h"
+#include "chrome/installer/util/install_service_work_item.h"
+#include "third_party/abseil-cpp/absl/cleanup/cleanup.h"
 
 namespace brave_vpn {
 
+// The service starts under system user so we save crashes to
+// %PROGRAMDATA%\BraveSoftware\{service name}\Crashpad
+base::FilePath GetVpnHelperServiceProfileDir() {
+  std::wstring program_data =
+      install_static::GetEnvironmentString(L"PROGRAMDATA");
```

Comment:
we don't need install_static here
```
DIR_COMMON_APP_DATA,         // Usually "C:\ProgramData". Data for specific
                               // applications is stored in subdirectories.
```

Reply:
I guess we still have https://github.com/brave/brave-core/pull/20754/files#diff-c9c98498ca4893ea43c148319bfcbdc7b78dbb665b3dc5acd5d57f8576a05db9R76 used in components though so this can be a follow-up